examples/landsat-c2-l1/LM01_L1GS_001010_19720908_02_T2/LM01_L1GS_001010_19720908_02_T2.json

```diff
@@ -7,7 +7,7 @@
     "instruments": [
       "mss"
     ],
-    "created": "2022-03-31T18:24:59.855547Z",
+    "created": "2022-10-15T12:22:11.833165Z",
```

Comment:
IMO not worth it for this PR, but I'm starting to think we _shouldn't_ include `created` in our example files, in order to cut down on diff noise. IMO `stac.create_item` should still set `created`, but we should either
- strip it off in a `create_examples`script, or
- provide a command-line flag to not include the `created` field

I'm dropping this note here because I've been thinking about it but haven't thought about it during a PR review until now.